File: Announcement.Core/Models/SingleReport.cs
using System;
using Newtonsoft.Json;

namespace Announcement.Core
{
    public class SingleReport 
    {
        [JsonProperty("latitude")]
        public float Latitude { get; set; }

        [JsonProperty("longitude")]
        public float Longitude { get; set; }

        [JsonProperty("photo")]
        public byte[] Photo { get; set; }
    }
}

File: workspace/api-stop-spam/src/main/java/api/SingleReport.java
package api;

import api.Report;
import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBIgnore;

public class SingleReport extends Report 
{
    protected byte[] photo;

    @DynamoDBIgnore
    public byte[] getPhoto() 
    {
        return this.photo;
    }

    public void setPhoto(byte[] photo) 
    {
        this.photo = photo;
    }

    public SingleReport() 
    {
    	
    }

    public SingleReport(float latitude, float longitude, byte[] photo) 
    {
        this.latitude = latitude;
        
        this.longitude = longitude;
        
        this.photo = photo;
    }
}